.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+* @KittyCAD/frontend
+/src/ @KittyCAD/frontend
+/rust/ @KittyCAD/frontend
```

Comment:
@jtran @adamchalmers Sounds like we need a new `@kittycad/kcl` or `@kittycad/language` team?

Reply:
I'd like a `language` team. But either one would be good.

Reply:
@jtran @KittyCAD/kcl isn't in this list https://github.com/KittyCAD/modeling-app/settings/access, maybe that's it?

Reply:
![image](https://github.com/user-attachments/assets/d4ca5b08-d710-4cac-9c1a-004445ca969d)